### ISSUE TITLE
goimports is default behavior

### ISSUE THREAD

**Issue by strogiyotec** [fatih/vim-go#2855]
Good day,  thank you for this amazing plugin. I'm going through Tutorial part and noticed this feature
```
If you haven't heard yet, it's called goimports. goimports is a replacement for gofmt. You have two ways of using it. The first (and recommended) way is telling vim-go to use it when saving the file:
let g:go_fmt_command = "goimports"
```
From the text above it's not clear that goimports is default behavior and it will be called every time when you save a file. Could I create a pool request with new text something like this ? 
```
The first (default and recommended) way is telling vim-go to use it when saving the file:
let g:go_fmt_command = "goimports"
``

**Comment by bhcleek** [comment]
It's only called every time when `g:go_fmt_autosave` is set. I'd accept a PR that makes that clear.

**Comment by strogiyotec** [comment]
@bcarrell  Thank you for quick response. With my configuration it works by default even without 
```
let g:go_fmt_command = "goimports"
```
in config file. Here is the my vimrc
```
"Vim go
 call plug#begin('~/.vim/plugged')
 Plug 'fatih/vim-go', { 'do': ':GoUpdateBinaries' }
 Plug 'AndrewRadev/splitjoin.vim'
 Plug 'ctrlpvim/ctrlp.vim'
 call plug#end()
  let g:go_highlight_functions = 1
  let g:go_auto_type_info = 1
  let g:go_rename_command = 'gopls'
```
This is the reason why I thought that **goimports** is enabled by default. Is it a problem in my conf ?

**Comment by bhcleek** [comment]
That's a bug that. I put up a PR to fix that: #2858

**Comment by strogiyotec** [comment]
Ok in this case there is no need to add this info to Tutorial , thank you , I will close the ticket
